CONTRIBUTING.md

```diff
@@ -1,25 +1,32 @@
 # Contributing to vagrant-service-manager Plugin
 
 The following is a set of guidelines for contributing to the
-`vagrant-service-manager` plugin, which is hosted in the [`Project Atomic
-Organization`](https://github.com/projectatomic) on GitHub.
+vagrant-service-manager plugin, which is hosted in the [Project Atomic
+Organization](https://github.com/projectatomic) on GitHub.
 
 These are just guidelines, please use your best judgement and feel free
 to propose changes to this document in a pull request.
 
 At this point, this document is not complete, but as decisions are made on the
-[`container-tools@redhat.com`](https://www.redhat.com/mailman/listinfo/container-tools)
+[container-tools@redhat.com](https://www.redhat.com/mailman/listinfo/container-tools)
 mailing list they will be added to this document.
 
-## Pull Requests
+
+## Submitting Issues
+
+You can submit issues with respect to the vagrant-service-manager plugin [here](https://github.com/projectatomic/vagrant-service-manager/issues/new).Make sure you include all the relevant details pertaining the issue.
+Before submitting new issues, it is suggested to check [all existing issues](https://github.com/projectatomic/vagrant-service-manager/issues) in order to avoid duplication.
+
+
```

Comment:
2 things:

1. Should we move this to the README or at least put in a pointer?

2. Should we add something like:

The vagrant service-manager plugin works closely with the Atomic Developer Bundle (ADB) and the adb-utils RPM.  You may wish to review the issues in both repos as well.

With appropriate links